hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/HapiTestEnv.java

```diff
@@ -69,11 +69,7 @@ public HapiTestEnv(@NonNull final String testName, final boolean cluster) {
                 final var workingDir = Path.of("./build/hapi-test/" + testName + "/node" + nodeId)
                         .normalize();
                 setupWorkingDirectory(workingDir, configText);
-                if (nodeId == 0) {
-                    nodes.add(new InProcessHapiTestNode(workingDir, 0, FIRST_GRPC_PORT));
-                } else {
-                    nodes.add(new SubProcessHapiTestNode(workingDir, nodeId, FIRST_GRPC_PORT + (nodeId * 2)));
-                }
+                nodes.add(new SubProcessHapiTestNode(workingDir, nodeId, FIRST_GRPC_PORT + (nodeId * 2)));
```

Comment:
This is obviously what was mentioned in the PR comment as "Avoids ISS in smallContractCreate() by removing special treatment of node0 in HapiTestEnv." - but ... the "InProcessHapiTestNode" is there for a reason (performance I think) ... is this the desired change?

Reply:
Not sure to be fair, wasn't my change, checkout this Slack [thread](https://limechain.slack.com/archives/C040H9JPQT1/p1696384945548229)

Reply:
The in process node supports debugging in an IDE.
It needs to be removed temporarily because it isn't playing nice with the out-of-process setup, but future work is needed to re-enable it in a way that supports IDE debugging.